lib/js/components/Layout/Layout.consts.ts

```diff
@@ -0,0 +1,22 @@
+export const LAYOUT_CONTENT_COLUMN_SIZE = {
+	FULL_WIDTH: 'fullWidth',
+	MAX_900: 'max900',
+} as const;
+
+export type LayoutContentColumnSize =
+	typeof LAYOUT_CONTENT_COLUMN_SIZE[keyof typeof LAYOUT_CONTENT_COLUMN_SIZE];
+export const LAYOUT_RIGHT_COLUMN_SIZE = {
+	MEDIUM: 'medium',
+	LARGE: 'large',
```

Comment:
IMO, `LARGE` looks like a separate layout, can you discuss with UX if we should expose this prop for 3 column layout?